x-pack/plugins/lens/public/xy_visualization/expression.tsx

```diff
@@ -125,6 +129,10 @@ export const xyChart: ExpressionFunctionDefinition<
         defaultMessage: 'Define how missing values are treated',
       }),
     },
+    valueLabels: {
```

Comment:
If it's just a simple enum, what about using
```
valueLabels: {
      types: ['string'],
      options: ['hide', 'show']
      help: '',
}
```

That would reduce complexity a little in a bunch of places because it's not necessary anymore to nest the expression an additional layer